phpstan.neon

```diff
@@ -5,6 +5,7 @@ parameters:
     level: 5
     paths:
         - %currentWorkingDirectory%/lib
+        - %currentWorkingDirectory%/tests/Doctrine/StaticAnalysis
```

Comment:
Let's get rid of `%currentWorkingDirectory%` instead? It does not bring any benefit.

Reply:
Done